roles/autotested/tasks/main.yml

```diff
@@ -32,6 +32,11 @@
     path: "{{ autotest_dest_dir }}/RELEASE-VERSION"
     state: absent
 
+- name: Bundled Docker Autotest directory is removed
+  file:
+    path: "{{ autotest_docker_path }}"
+    state: absent
+
```

Comment:
Unf. the clone happens in Jenkins before ADEPT is even called.  That's needed so that the pull-request triggering and change-tracking can work.

However, what you have above would be a great fall-back to add in, e.g. when cache (jenkins repo clone) isn't available.  That would let ADEPT run more stand-alone, not requiring pre-cloned autotest/docker autotest repos.